Microsoft.Toolkit.Uwp.UI.Controls/HeaderedTextBlock/HeaderedTextBlock.Properties.cs

```diff
@@ -9,6 +9,15 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     public partial class HeaderedTextBlock
     {
         /// <summary>
+        /// Defines the <see cref="TextFontSize"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty TextFontSizeProperty = DependencyProperty.Register(
+            nameof(TextFontSize),
+            typeof(int),
+            typeof(HeaderedTextBlock),
+            new PropertyMetadata(18));
```

Comment:
As this is set in the default style, you shouldn't be setting it here too, I would suggest default(int)